itest/lnd_sweep_test.go

```diff
@@ -2352,3 +2352,177 @@ func testFeeReplacement(ht *lntest.HarnessTest) {
 	// Finally, clean the mempool.
 	ht.MineBlocksAndAssertNumTxes(1, 1)
 }
+
+// testBumpFeeLowBudget checks that when the requested ideal budget cannot be
```

Comment:
Nice test, I wonder if we can also test a case where we restart and a new sweepInput is available which would be added to the retryGroup ?

Reply:
I don't think that's needed - you are essentially testing the restart behavior of the wallet utxo management, which should have its own test and it's unrelated to this PR. In addition the restart behavior for the sweeper has been tested in the channel force close tests.